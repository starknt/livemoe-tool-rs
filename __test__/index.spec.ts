import { beforeAll, expect } from 'vitest'
import { test } from 'vitest'

import { hideDesktopIcon, setWindowWorker, restoreWindowWorker } from '../index'

test('', () => {
  expect(restoreWindowWorker()).toMatchSnapshot()
  expect(hideDesktopIcon()).toMatchInlineSnapshot('undefined')
})