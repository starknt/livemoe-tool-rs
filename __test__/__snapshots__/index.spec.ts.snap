// Vitest Snapshot v1

exports[` 1`] = `undefined`;
